setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
I think it'd be useful to get fedmsg into the semver phase where the X number means the API is changing - that's a nice handy way to indicate to the users that they need to watch out ☺

0.19 sounds fine to me.